scenario/ApproveThisScenario.ts

```diff
@@ -1,18 +1,15 @@
 import { scenario } from './context/CometContext';
 import { expect } from 'chai';
 import { constants } from 'ethers';
-import { setNextBaseFeeToZero } from './utils';
 
 scenario('Comet#approveThis > allows governor to authorize and rescind authorization for Comet ERC20', {}, async ({ comet, timelock, actors }, world, context) => {
   const { admin } = actors;
 
-  await setNextBaseFeeToZero(world);
-  await admin.approveThis(timelock.address, comet.address, constants.MaxUint256, { gasPrice: 0 });
+  await admin.approveThis(timelock.address, comet.address, constants.MaxUint256, world);
```

Comment:
Damn going to increase my conflicts 😆 I'm moving world into context so we don't need to pass it around everywhere, since it's also error prone and strictly tied to it

Reply:
Doh! Should I revert this change? I can do the re-factor after your changes are in.

Reply:
Reverted the change.